### DISCUSSION TITLE
my school unblocked crosh, what can i do?

### DISCUSSION THREAD

**Discussion by brobcoolguy** [3kh0/ext-remover#909] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
dev mode is blocked btw.

**Comment by AshtonDavies** [comment, marked as answer]
OP Crosh: Check the README.

**Reply by BrodXI** [reply]
I cant find OP Crosh

**Reply by funkymokey** [reply]
3kh0 needs to bring back the old UI this new one is unreadable

**Reply by AshtonDavies** [reply]
Looks like OP Crosh has been removed from the README. @Brandon421-ops 's fork got deleted for some weird reason, but I should probably add it to my site soon because I've heard from him it actually does disable extensions.

**Reply by AshtonDavies** [reply]
@rodplayz Instructions are now available [here](https://chrose.netlify.app/detail/crosh%20&%20vmc%20disable%20extensions) if you want to try the exploit. I forgot that it was never added to this repo.

**Reply by ZXMushroom63** [reply]
With op crosh, can you corrupt policies?
`vmc create-extra-disk --size 1 /etc/opt/chrome/policies`
Or does the extension bypass only work if it isn't installed yet?

**Reply by AshtonDavies** [reply]
> With op crosh, can you corrupt policies?
> `vmc create-extra-disk --size 1 /etc/opt/chrome/policies`
> Or does the extension bypass only work if it isn't installed yet?

For disabling extensions, it screws around with the disk size I think, so I guess you can't corrupt policies with Crosh.

**Comment by cloneyfirey** [comment]
yay
